packages/lodestar/src/chain/attestation.ts

```diff
@@ -79,9 +84,17 @@ export class AttestationProcessor implements IAttestationProcessor {
     const target = attestation.data.target;
     assert([currentEpoch, previousEpoch].includes(target.epoch));
     assert(target.epoch === computeEpochAtSlot(this.config, attestation.data.slot));
+    assert(
+      getCurrentSlot(this.config, checkpointState.genesisTime) >= computeStartSlotAtEpoch(this.config, target.epoch)
+      , "Current slot less than this target epoch's start slot"
+    );
     const block = await this.db.block.get(attestation.data.beaconBlockRoot);
-    assert(block.message.slot <= attestation.data.slot);
+    assert(block.message.slot <= attestation.data.slot, "Attestation is for past block");
 
+    assert(
+      getCurrentSlot(this.config, checkpointState.genesisTime) >= attestation.data.slot + 1,
+      "Attestation slot is not in the past"
+    );
```

Comment:
I don't think we should have this check as because of time diff some nodes might be ahead of us

Reply:
This check also created codeclimate issue :smile:

Reply:
Didn't even think about that. Maybe we should bring that up in the spec. Pushing a fix right now.